simphony/cuds/lattice.py

```diff
@@ -149,17 +151,18 @@ def data(self, value):
 
 
 def make_hexagonal_lattice(name, h, size, origin=(0, 0)):
-    """Create and return a 2D hexagonal lattice.
+    """Create and return a 2D hexagonal lattice embedded on the XY-plane
```

Comment:
As I understood the SSB statement, we only support "2D lattice" on an XY-plane. Personally, I would have allowed all planar configurations, but it's simpler this way anyway. Also, one can always make a 3D lattice with size dimension 1 to any direction to create fully working planar 2D lattice.